code/modules/hydroponics/grown/olive.dm

```diff
@@ -12,7 +12,7 @@
 	growthstages = 6
 	growing_icon = 'icons/obj/hydroponics/growing_fruits.dmi'
 	genes = list(/datum/plant_gene/trait/repeated_harvest)
-	mutatelist = list(/obj/item/seeds/apple)
+	mutatelist = list()
```

Comment:
```suggestion
```

Reply:
IMO, I think its fine to leave it explicit in this case

Reply:
Fine with me aswell

Reply:
I mirror'd other plants that don't have any other mutations when changing it like this.